integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -908,6 +929,22 @@ fn get_base_shard_layout(version: u64) -> ShardLayout {
     }
 }
 
+fn account_exist_in_view_client(
+    env: &mut TestLoopEnv,
+    node_index: usize,
+    account_id: AccountId,
+    height: u64,
+) -> bool {
+    let view_client_handle = env.datas[node_index].view_client_sender.actor_handle();
+    let view_client = env.test_loop.data.get_mut(&view_client_handle);
+    let msg = Query::new(
+        BlockReference::BlockId(BlockId::Height(height)),
+        QueryRequest::ViewAccount { account_id },
+    );
+    let result = near_async::messaging::Handler::handle(view_client, msg);
+    result.is_ok()
```

Comment:
Is this sufficient to check that it returned ok?

Reply:
Yes, if the result is `Result<QueryResponse, QueryError>`
and `QueryResponse` contains a data of a succesful response.